test/performance/perf-upsert/src/main.rs

```diff
@@ -32,6 +32,7 @@ async fn main() {
     if let Err(e) = run().await {
         eprintln!("ERROR: {}", e);
         // TODO(rkhaitan) could this be simpler
+        // I doubt it
```

Comment:
The rendering of the error, you mean? If so, just use the debug implementation (`eprintln!("ERROR: {:?}, e)`) and anyhow will do the hard work of rendering the causes: https://docs.rs/anyhow/1.0.31/anyhow/struct.Error.html#display-representations.

Reply:
was just a fake comment to get this test to run lol but thank you!

Reply:
Lol yeah sorry, I meant the original todo! I just happened to notice it because of your comment